charts/datadog/README.md

```diff
@@ -490,6 +490,8 @@ helm install <RELEASE_NAME> \
 | clusterAgent.admissionController.failurePolicy | string | `"Ignore"` | Set the failure policy for dynamic admission control.' |
 | clusterAgent.admissionController.mutateUnlabelled | bool | `false` | Enable injecting config without having the pod label 'admission.datadoghq.com/enabled="true"' |
 | clusterAgent.admissionController.remoteInstrumentation.enabled | bool | `false` | Enable polling and applying library injection using Remote Config. # This feature is in beta, and enables Remote Config in the Cluster Agent. It also requires Cluster Agent version 7.43+. # Enabling this feature grants the Cluster Agent the permissions to patch Deployment objects in the cluster. |
+| clusterAgent.admissionController.service.port | int | `8000` | Set port of cluster-agent admission controller service |
+| clusterAgent.admissionController.service.type | string | `"ClusterIP"` | Set type of cluster-agent admission controller service |
```

Comment:
I don't this we need to have the service type configurable for the admission controller

Reply:
@clamoriniere 
That makes sense. Please confirm that I have made the correction to use only `admissionController.port`